pkg/database/migrations.go

```diff
@@ -1393,6 +1393,32 @@ func (db *Database) getMigrations(ctx context.Context) *gormigrate.Gormigrate {
 				return tx.Exec(`DROP TABLE audit_entries`).Error
 			},
 		},
+		{
+			ID: "00056-AuthorzedAppsAPIKeyTypeBasis",
+			Migrate: func(tx *gorm.DB) error {
+				if err := tx.AutoMigrate(&AuditEntry{}).Error; err != nil {
+					return err
+				}
+
+				sqls := []string{
+					`UPDATE authorized_apps ALTER COLUMN api_key_type DROP DEFAULT`,
+					`UPDATE authorized_apps ALTER COLUMN api_key_type SET NOT NULL`,
+					`UPDATE authorized_apps SET api_key_type = 2 WHERE api_key_type = 1`,
```

Comment:
I'm opposed to changing the values of these live. I'm worry about outages or inconsistencies

Reply:
Updated